psamm_import/importer.py

```diff
@@ -123,6 +123,33 @@ def _decimal_representer(dumper, data):
     return dumper.represent_scalar('tag:yaml.org,2002:float', value)
 
 
+def default_compartment(model):
```

Comment:
In this case, I think the first word in the function name should be a verb, maybe "detect" or just "get" (e.g. `detect_default_compartment()`).